gallery.js

```diff
@@ -2,12 +2,19 @@ let fs = require('fs');
 
 let imageFileNames = fs.readdirSync('site/images');
 
-// This code is just here to demonstrate how fs.readdirSync works
-
-console.log('sites/images contains the following files:')
+//This code is just here to demonstrate how fs.readdirSync works
 
+console.log('sites/images contains the following files:');
+let imageGallery;
 for (let i = 0; i < imageFileNames.length; i++) {
   let fileName = imageFileNames[i];
-
-  console.log(`${i + 1}. ${fileName}`);
+  if (i === 0) {
+    imageGallery = `<img src="images/${fileName}" height="400" width="550">`;
+  }
+  else {
+    imageGallery = imageGallery + `<img src="images/${fileName}" height="400"width="550" >`;
```

Comment:
You probably need a space between `height="400"width="550"`. I'm not sure if the browser will handle that.